app/assets/stylesheets/article-show.scss

```diff
@@ -596,7 +582,17 @@ article {
     }
 
     kbd {
-      @extend %kbd;
+      display: inline-block;
+      padding: 3px 5px;
+      font-size: 11px;
+      line-height: 10px;
+      color: #444d56;
+      vertical-align: middle;
+      background-color: #fafbfc;
+      border: solid 1px #c6cbd1;
```

Comment:
I don't know what `kbd`, but any reason why we aren't using the new defined system colors here?

Reply:
`kbd` is an HTML element representing keyboard keys.. Technically should be used for example when you want to show some keyboard shortcut in your post :) 

To be clear, I didn't come up with this code, I just moved it from other place in our codebase.
Also, as mentioned on Slack, the colors I've just added to the codebase are not really imported anywhere in the codebase yet - I'm only preparing ground for now... Once we're in good place I'll start replacing HEX values with some of our variables. But there's more that needs to be done first.

Reply:
👍🏽